### ISSUE TITLE
Ambiguous use of "filtered"

### ISSUE THREAD

**Issue by iconara** [hashicorp/nomad#4471]
When I query an evaluation, or a job, that has not been possible to place I get a list that uses the word "filtered" ambiguously.

For example

```
Placement Failure
Task Group "foo":
  * Class "class1" filtered 1 nodes
  * Class "class2" filtered 3 nodes
  * Class "class3" filtered 1 nodes
  * Constraint "${meta.role} = role1" filtered 5 nodes
```

I can't tell whether or not "filtered" in this list means "selected" or "rejected". I assume "rejected" because I can kind of figure that out from the last list entry and my knowledge about the context, but at the same time, in most programming languages, including Go, that have a function called something like "filter" it means "select", so that's what I think when I read the list.

Would you consider changing the wording in this listing to be less ambiguous? I suggest using "rejected".

**Comment by preetapan** [comment]
@iconara thanks for the suggestion. While I can see how generally filter can mean both "filter out" and "filter only", in this specific case the intent is "filtered out" and hopefully known from the rest of the sentence. It is listing specific constraints and showing how many nodes did not meet the constraint. 

We could perhaps change it to "filtered out" to make it clearer.

**Comment by lgfa29** [comment]
Closed by #7369
